tests/host-recovery-source.toit

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2024 Toitware ApS.
+
+import artemis-pkg.artemis as artemis
+import host.os
+import http
+import net
+import system.storage
+
+TEST-URL-ENV ::= "TEST-URL"
+
+main:
+  client := http.Client net.open
```

Comment:
This should all go away, right? Add a TODO comment.

Reply:
I think it's OK to leave. So that the test and the test-device have a way of synchronizing.
We only remove the old broker once we receive an http request.